src/util/initEvents.js

```diff
@@ -9,7 +9,7 @@ export const initEvents = () => {
   // prevent browser from restoring the scroll position so that we can do it manually
   window.history.scrollRestoration = 'manual'
 
-  window.addEventListener('keydown', handleKeyboard)
```

Comment:
Handlers have to be bound to some element/component (not window or document) or tests have no way of using them. I'll look into Search and New Thought soon.

Reply:
It might be feasible to attach `handleKeyboard` to a component for testing purposes while attaching it to `window` during normal execution of the app. 

In the tests, we can always dispatch the action directly (e.g. `{ type: 'search', value: 'TEXT' }`) rather than simulating the event. 

Note that we should *not* do anything specific for Search and New Thought, but rather have a general solution.